### PR TITLE
fix: repo URLs

### PR DESCRIPTION
**Pull request by alanshaw**
Fixes a _lot_ of old repo URLs e.g.`w3s-project` and `web3.storage`

Needs a follow up PR to remove references to reference docs site that does not exist: `web3-storage.github.io`

I'd like to use [jsr](https://jsr.io/) for this somehow!

### CODE REVIEW

File: packages/access-client/readme.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ async function createAgent() {
 }
 ```
 
-On node.js, use [`StoreConf`](https://web3-storage.github.io/w3protocol/classes/_web3_storage_access.StoreConf.html), which uses the [`conf` package](https://www.npmjs.com/package/conf) to store keys and metadata in the user's platform-specific default configuration location (usually in their home directory).
+On node.js, use [`StoreConf`](https://web3-storage.github.io/w3up/classes/_web3_storage_access.StoreConf.html), which uses the [`conf` package](https://www.npmjs.com/package/conf) to store keys and metadata in the user's platform-specific default configuration location (usually in their home directory).
```
Comment (fforbeck) [inline packages/access-client/readme.md:61]: Got 404

File: packages/access-client/readme.md [hunk 2]
```diff
@@ -91,13 +91,13 @@ async function createAgent() {
 
 ```
 
-See the [`AgentCreateOptions` reference](https://web3-storage.github.io/w3protocol/interfaces/_web3_storage_access._internal_.AgentCreateOptions.html) if you want to configure the agent to use a non-production service connection.
+See the [`AgentCreateOptions` reference](https://web3-storage.github.io/w3up/interfaces/_web3_storage_access._internal_.AgentCreateOptions.html) if you want to configure the agent to use a non-production service connection.
```
Comment (fforbeck) [inline packages/access-client/readme.md:94]: 404

File: packages/access-client/readme.md [hunk 3]
```diff
@@ -91,13 +91,13 @@ async function createAgent() {
...
 
 ### Space creation
 
 A newly-created agent does not have access to any spaces, and is thus unable to store data using the w3up platform.
 
-To create a new space, use [`agent.createSpace`](https://web3-storage.github.io/w3protocol/classes/_web3_storage_access.Agent.html#createSpace), optionally passing in a human-readable name.
+To create a new space, use [`agent.createSpace`](https://web3-storage.github.io/w3up/classes/_web3_storage_access.Agent.html#createSpace), optionally passing in a human-readable name.
```
Comment (fforbeck) [inline packages/access-client/readme.md:100]: 404

File: packages/access-client/readme.md [hunk 4]
```diff
@@ -126,21 +126,21 @@ interface CreateSpaceResult {
...
 
-Agents may also have a "current" space, which is used as the default space for storage operations if none is specified. You can retrieve the DID of the current space with [`agent.currentSpace`](https://web3-storage.github.io/w3protocol/classes/_web3_storage_access.Agent.html#currentSpace). If you also want the metadata and proofs associated with the space, use [`agent.currentSpaceWithMeta`](https://web3-storage.github.io/w3protocol/classes/_web3_storage_access.Agent.html#currentSpaceWithMeta).
+Agents may also have a "current" space, which is used as the default space for storage operations if none is specified. You can retrieve the DID of the current space with [`agent.currentSpace`](https://web3-storage.github.io/w3up/classes/_web3_storage_access.Agent.html#currentSpace). If you also want the metadata and proofs associated with the space, use [`agent.currentSpaceWithMeta`](https://web3-storage.github.io/w3up/classes/_web3_storage_access.Agent.html#currentSpaceWithMeta).
```
Comment (fforbeck) [inline packages/access-client/readme.md:129]: 404

File: packages/access-client/readme.md [hunk 5]
```diff
@@ -126,21 +126,21 @@ interface CreateSpaceResult {
...
 
-To set the current space, use [`agent.setCurrentSpace`](https://web3-storage.github.io/w3protocol/classes/_web3_storage_access.Agent.html#setCurrentSpace). Note that this must be done explicitly; creating an agent's first space does not automatically set it as the current space.
+To set the current space, use [`agent.setCurrentSpace`](https://web3-storage.github.io/w3up/classes/_web3_storage_access.Agent.html#setCurrentSpace). Note that this must be done explicitly; creating an agent's first space does not automatically set it as the current space.
```
Comment (fforbeck) [inline packages/access-client/readme.md:131]: 404

File: packages/access-client/readme.md [hunk 6]
```diff
@@ -126,21 +126,21 @@ interface CreateSpaceResult {
...
 
 ### Space registration
 
 A newly-created space must be registered with the w3up access service before it can be used as a storage location.
 
-To register a space, use [`agent.registerSpace`](https://web3-storage.github.io/w3protocol/classes/_web3_storage_access.Agent.html#registerSpace), which takes an email address parameter and registers the [current space](#managing-the-current-space) with the access service.
+To register a space, use [`agent.registerSpace`](https://web3-storage.github.io/w3up/classes/_web3_storage_access.Agent.html#registerSpace), which takes an email address parameter and registers the [current space](#managing-the-current-space) with the access service.
```
Comment (fforbeck) [inline packages/access-client/readme.md:137]: 404

File: packages/access-client/readme.md [hunk 7]
```diff
@@ -126,21 +126,21 @@ interface CreateSpaceResult {
...
 
 Calling `registerSpace` will cause the access service to send a confirmation email to the provided email address. When the activation link in the email is clicked, the service will send the agent a delegation via a WebSocket connection that grants access to the services included in w3up's free tier. The `registerSpace` method returns a `Promise` that resolves once the registration process is complete. Make sure to wrap calls to `registerSpace` in a `try/catch` block, as registration will fail if the user does not confirm the email (or if network issues arise, etc.).
 
 ### Delegating to another agent
 
-The agent's [`delegate` method](https://web3-storage.github.io/w3protocol/classes/_web3_storage_access.Agent.html#delegate) allows you to delegate capabilities to another agent.
+The agent's [`delegate` method](https://web3-storage.github.io/w3up/classes/_web3_storage_access.Agent.html#delegate) allows you to delegate capabilities to another agent.
```
Comment (fforbeck) [inline packages/access-client/readme.md:143]: 404

File: packages/access-client/readme.md [hunk 8]
```diff
@@ -158,15 +158,15 @@ Note that the receiving agent will need to [import the delegation](#importing-de
 
 ### Importing delegations from another agent
 
-The [`addProof` method](https://web3-storage.github.io/w3protocol/classes/_web3_storage_access.Agent.html#addProof) takes in a ucanto `Delegation` and adds it to the agent's state Store. The proof of delegation can be retrieved using the agent's [`proofs` method](https://web3-storage.github.io/w3protocol/classes/_web3_storage_access.Agent.html#proofs).
+The [`addProof` method](https://web3-storage.github.io/w3up/classes/_web3_storage_access.Agent.html#addProof) takes in a ucanto `Delegation` and adds it to the agent's state Store. The proof of delegation can be retrieved using the agent's [`proofs` method](https://web3-storage.github.io/w3up/classes/_web3_storage_access.Agent.html#proofs).
```
Comment (fforbeck) [inline packages/access-client/readme.md:161]: 404

File: packages/access-client/readme.md [hunk 9]
```diff
@@ -158,15 +158,15 @@ Note that the receiving agent will need to [import the delegation](#importing-de
...
+The [`importSpaceFromDelegation` method](https://web3-storage.github.io/w3up/classes/_web3_storage_access.Agent.html#importSpaceFromDelegation) also accepts a ucanto `Delegation`, but it is tailored for "full delegation" of all space-related capabilities. The delegated ability must be `*`, which is the "top" ability that can derive all abilities for the Space's DID. Use `importSpaceFromDelegation` in preference to `addProofs` when importing a full `*` delegation for a space, as it also adds metadata about the imported space to the Agent's persistent store and adds the space to the agent's set of authorized spaces.
```
Comment (fforbeck) [inline packages/access-client/readme.md:163]: 404

File: packages/filecoin-client/README.md [hunk 10]
```diff
@@ -3,7 +3,7 @@
 
 ## About
 
-The `@web3-storage/filecoin-client` package provides the "low level" client API to make data uploaded with the w3up platform available in Filecoin Storage providers. It is based on [web3-storage/specs/w3-filecoin.md](https://github.com/web3-storage/specs/blob/feat/filecoin-spec/w3-filecoin.md) and is not intended for web3.storage end users.
+The `@web3-storage/filecoin-client` package provides the "low level" client API to make data uploaded with the w3up platform available in Filecoin Storage providers. It is based on [web3-storage/specs/w3-filecoin.md](https://github.com/storacha/specs/blob/feat/filecoin-spec/w3-filecoin.md) and is not intended for web3.storage end users.
```
Comment (fforbeck) [inline packages/filecoin-client/README.md:6]: 404

File: packages/w3up-client/README.md [hunk 11]
```diff
@@ -1,10 +1,10 @@
...
+  <a href="https://github.com/storacha/w3up/actions/workflows/w3up-client.yml"><img alt="GitHub Workflow Status" src="https://img.shields.io/github/actions/workflow/status/storacha/w3up/w3up-client.yml?branch=main&style=for-the-badge" /></a>
   <a href="https://discord.com/channels/806902334369824788/864892166470893588"><img src="https://img.shields.io/badge/chat-discord?style=for-the-badge&logo=discord&label=discord&logoColor=ffffff&color=7389D8" /></a>
   <a href="https://x.com/web3storage"><img alt="X Follow" src="https://img.shields.io/badge/follow-web3storage-black?style=for-the-badge&logo=x" /></a>
-  <a href="https://github.com/web3-storage/w3up-client/blob/main/LICENSE.md"><img alt="License: Apache-2.0 OR MIT" src="https://img.shields.io/badge/LICENSE-Apache--2.0%20OR%20MIT-yellow?style=for-the-badge" /></a>
+  <a href="https://github.com/storacha/w3up-client/blob/main/LICENSE.md"><img alt="License: Apache-2.0 OR MIT" src="https://img.shields.io/badge/LICENSE-Apache--2.0%20OR%20MIT-yellow?style=for-the-badge" /></a>
```
Comment (fforbeck) [inline packages/w3up-client/README.md:7]: 404

**Review by fforbeck — COMMENTED** [review]
Got several broken links (404). I believe it would be nice to have them pointing to the right place or even remove the link ref if needed.

**Comment by alanshaw** [comment]
Most of those are to the github.io site that I mentioned in the PR description...I should have just left them alone 🤷‍♂️

The other two I will fix - thanks for being diligent ❤️

**Review by fforbeck — APPROVED** [review]

**Pull request merged**